content/open-source/wolfi/wolfi-with-dockerfiles.md

```diff
@@ -21,20 +21,16 @@ toc: true
 specifically for cloud workloads, with an emphasis on software supply chain security. Using
 [apk](https://wiki.alpinelinux.org/wiki/Package_management) for package management, Wolfi differs
 from Alpine in a few important aspects, most notably the use of glibc instead of musl and the fact
-that Wolfi doesn't have a kernel as it is inteded to be used with a container runtime. The minimal
-footprint makes Wolfi an ideal base for both _distroless_ images and fully-featured builder
-images.
+that Wolfi doesn't have a kernel as it is intended to be used with a container runtime. The minimal
+footprint makes Wolfi an ideal base for both _distroless_ images and fully-featured builder images.
 
-A _distroless_ image is a minimal container image that typically doesn't include a shell or package manager. The extra tightness improves security in several aspects, but it requires a more sophisticated strategy for image composition since you can't install packages so easily.
+A distroless image is a minimal container image that typically doesn't include a shell or package manager. The extra tightness improves security in several aspects, but it requires a more sophisticated strategy for image composition since you can't install packages so easily. You can learn more about distroless in our [Going Distroless](https://edu.chainguard.dev/chainguard/chainguard-images/getting-started-distroless/) guide.
 
-There are currently two main strategies for building distroless images with Wolfi:
+Wolfi-based builder images are still a better and more secure option to use as base images in your Dockerfile than using a full-fledged Linux distribution, as they are smaller and have fewer CVEs.
```

Comment:
I would add this sentence to the previous paragraph, before the "You can learn more..." sentence. Your call though!